README.md

```diff
@@ -1,47 +1,52 @@
-# W3C Trace Context
+# W3C's Trace Context standard and how to enrich message tracing with APM tools
 
-Here you're gonna find a simple description of what is the purpose of [W3C Trace Context](https://www.w3.org/TR/trace-context/) and what kind of problem it came to solve.
-
-## Simple Stack Trace
+In the software development process, when a system experiences a failure in runtime, it's natural for a developer to try to link that failure with who called that method and also which was the original request. This is where [stack trace](https://en.wikipedia.org/wiki/Stack_trace) comes in, look at an example below:
 
 ``` bash
 Unhandled exception. System.InvalidOperationException: Stack trace example
    at Program.CallChildActivity() in /Users/luizhlelis/Documents/projects/personal/trace-context-w3c/src/system-diagnostics-activity/Program.cs:line 22
    at Program.Main() in /Users/luizhlelis/Documents/projects/personal/trace-context-w3c/src/system-diagnostics-activity/Program.cs:line 11
 ```
 
-![Distributed Trace](doc/distributed-trace.png)
+with the information above we're able to know that a failure happened at `CallChildActivity()` method (line 22) which was called by the `Main()` method (line 11) both inside the `Program.cs` class. That's the reason why the runtime message tracking is essential for a software health and reliability, a rich information like that greatly increases the troubleshoot productivity. So the `stack trace` serves very well in terms of message trace when the subject is a single process application, like monolithic systems. On the other hand, when dealing with distributed systems like in a microservice architecture, the stack trace is not enough to expose the entire message tracking, that's the reason why distributed tracing tools and standards became necessary. The W3C defines a standard for this type of tracking, which is called `Trace Context`.
```

Comment:
**suggestion**: use suits instead of serves

```
So the `stack trace` suits very well in terms
```

Reply:
nice **suggestion**, thanks!